src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -266,8 +270,7 @@ public override IEnumerable<DownloadClientItem> GetItems()
                         break;
 
                     case "stalledDL": // torrent is being downloaded, but no connection were made
-                        item.Status = DownloadItemStatus.Warning;
-                        item.Message = "The download is stalled with no connections";
+                        _stalledPolicy.Apply(torrent, item, this.Settings);
```

Comment:
More a question for my benefit, but at what point does qBit place a `stalledDL` status on a torrent?

Reply:
Let me know if this is enough: https://github.com/qbittorrent/qBittorrent/issues/7620#issuecomment-587797380

Otherwise, I am happy to dig around a bit more, maybe on their sources.

Reply:
From my experience is when a download does not have any connections or if there is an issue with the host connectivity.

Reply:
> A torrent is also marked as stale for a short period of time after it is rechecked and iirc after it is resumed from a paused state.

This concerns me a little as I believe it could mean that if the setting isn't high enough it could be removed on a restart of QBit etc.